dusty/cli/__init__.py

```diff
@@ -96,6 +98,12 @@ def main():
     if '-d' in sys.argv:
         return run_daemon()
 
+    if '-v' in sys.argv:
```

Comment:
but what if the user wants to run only `dusty -v`? Basically what I was trying to avoid is adding another command specifically for this